app/i18n/de/sub.php

```diff
@@ -35,6 +35,10 @@
 		'clear_cache' => 'Nicht cachen (für defekte Feeds)',
 		'css_help' => 'Ruft gekürzte RSS-Feeds ab (Achtung, benötigt mehr Zeit!)',
 		'css_path' => 'Pfad zur CSS-Datei des Artikels auf der Original-Webseite',
+		'proxy' => 'Verwende einen Proxy, um den Feed abzuholen', 
+		'proxy_help' => 'Wähle ein Protokoll (z.B. SOCKS5) und einen Proxy mit Port (z.B. 127.0.0.1:1080)',
+		'css_cookie' => 'Verwende Cookies beim Herunterladen des Feed-Inhalts mit CSS-Filtern',
+		'css_cookie_help' => 'Beispiel: foo=bar; gdpr_consent=true; cookie=value',
```

Comment:
To add keys, use the Makefile. It will save you some work. You've updated only 3 languages, the others needs to be updated as well.
On the CLI, you just need to run for instance:
```
make i18n-add-key key='sub.feed.proxy' value='Set a proxy for fetching this feed'
```
It will work only if the key is not set. So you'll need to remove your changes in the i18n files then run the command once for each key. Then you'll add your translations again.
The command will take care of ordering the keys, adding the keys in every files, add the todo flag on languages you cannot translate.

Reply:
Thanks. Never seen docker used in a Makefile before 😄

Reply:
It's very handy :)